.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode/
+samples/dotnetcore/image-transfer/ContainerRegistryTransfer/obj/
```

Comment:
If you are adding this use this for reference - https://github.com/github/gitignore/blob/master/VisualStudio.gitignore#L31